app/src/main/java/com/kamron/pogoiv/Pokefly.java

```diff
@@ -323,6 +325,10 @@ public int onStartCommand(Intent intent, int flags, int startId) {
         return START_REDELIVER_INTENT;
     }
 
+    private boolean isMovesetEnabled() {
```

Comment:
This new method can be avoided if possible, I'd prefer using `GoIVSettings.getInstance(this).isMovesetEnabled()` directly!